css/style.css

```diff
@@ -9,400 +14,622 @@ body {
+  width: 38px;
+  height: 38px;
+}
+
+.color-wight {
+  background: #DED;
+}
+
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000;
+}
+
+.button-minus,
+.button-plus {
+  border: none;
+}
+
+.button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #fff;
+}
+
+.bag__link {
+  position: absolute;
+  width: 299px;
+  height: 58px;
+  margin-left: 884px;
+  top: 1663px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000;
+}
+
+.bag__arrow {
+  position: absolute;
+  width: 241px;
+  left: 1407px;
+  top: 1665px;
+}
 
 /* Медиа запросы */
 
-@media (max-width: 1270px) { 
+@media (max-width: 3000px) {
+   .new-сollection__social {
+    width: 50%;
+    }
+}   
+
+@media (max-width: 1500px) {
+   .new-сollection__social {
+    width: 45%;
+    }
+} 
+
+@media (max-width: 1500px) {
    .title {
-       top: 30%;
-       width: 20%;
-       font-size: 50px;
-       line-height: 52px;
-       left: 60%;
+    top: 30%;
+    width: 20%;
+    font-size: 50px;
+    line-height: 52px;
+    left: 55%;
+    }
+} 
+ 
+@media (max-width: 1500px) { 
+   .text {
+    width: 35%;
+    left: 55%;
+    height: 60%;
+     }
+}
+ 
+@media (max-width: 1500px) { 
+   .new-сollection__main-card {
+    width: 38%;
+    left: 55%;
+    margin-top: 1%;
+     }
+}
+     
+@media (max-width: 1500px) { 
+   .girl {
+    width: 50%;
+    right: 60%;
+     }
+}
+
+@media (max-width: 1500px) {
+   .new-сollection__social {
+    width: 45.5%;
    }
-    } 
+}
+
+@media (max-width: 1300px) {
+   .new-сollection__social {
+    width: 47%;
+   }
+}
+
+@media (min-width: 1270px) { 
+   .body {
+    margin: 0 auto;
+   }
+}
 
-@media (max-width: 1270px) { 
-    .text {
-        width: 35%;
-        left: 60%;
-        height: 60%;
+@media (max-width: 1200px) { 
+   .new-сollection__social {
+    width: 45%;
     }
 }
 
-@media (max-width: 1270px) { 
-    .new-сollection__main-card {
-        width: 38%;
-        left: 60%;
-        margin-top: 5%;
+@media (max-width: 1200px) { 
+   .girl {
+    width: 45%;
+    right: 60%;
     }
 }
-    
-@media (max-width: 1270px) { 
-    .girl {
-        width: 50%;
-        margin-left: 5%;
+ 
+@media (max-width: 1200px) { 
+   .new-сollection__main-button {
+    width: 35%;
+    font-size: 8px;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+   .new-сollection__main-link {
+    width: 50%;
+    margin-left: 3%;
+    font-size: 10px;
+    margin-right: -15%;
+     }
+}
+ 
+@media (max-width: 1200px) { 
+   .new-сollection-arrow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)